casepro/contacts/models.py

```diff
@@ -430,7 +440,7 @@ def as_json(self, full=True):
         """
         Prepares a contact for JSON serialization
         """
-        result = {'id': self.pk, 'name': self.get_display_name()}
+        result = {'id': self.pk, 'name': self.get_display_name(), 'urns': self.get_display_urns()}
```

Comment:
The case read page fetches the full contact in a separate request, no? https://github.com/rapidpro/casepro/blob/8db6ef3db6736697d46d47b62ee2e2a517ea034d/static/coffee/controllers.coffee#L587-L587